### PR TITLE
remove diskless assignments on Unmount()

### LINKED ISSUE #5: remove diskless assignments on Umount()
So far diskless assignments get created "on demand" when needed. Our other plugins also delete diskless assignments when they are not needed anymore. It is not really a requirement for this plugin, but it is a good idea in general where we have like 10 hypervisor nodes where only one at a time accesses the data. There we really do not want to keep track of 9 unused diskless clients.

### PR DESCRIPTION
**Pull request by rck**
There is no need to keep these, it just complicates things.

Closes #5

### CODE REVIEW

File: driver.go [hunk 1]
```diff
@@ -390,3 +386,51 @@ func (l *LinstorDriver) toDisklessCreate(name, node string, params *LinstorParam
...
+	}
+	ctx := context.Background()
+
+	// view to get storage information as well
+	resources, err := c.Resources.GetResourceView(ctx, &lopt)
+	if err != nil {
+		return false, err
+	}
+	if len(resources) != 1 {
+		return false, errors.New("Resource filter has to contain exaclty one resource")
```
Comment (beornf) [inline driver.go]: Small typo: s/exaclty/exactly.

File: driver.go [hunk 2]
```diff
@@ -390,3 +386,51 @@ func (l *LinstorDriver) toDisklessCreate(name, node string, params *LinstorParam
...
+	}
+	if len(resources[0].Volumes) != 1 {
+		return false, errors.New("There has to be exaclty one one volume in the resource")
```
Comment (beornf) [inline driver.go]: s/exaclty/exactly and double "one"

**Comment by rck** [comment]
Thanks, for the review. force pushed

**Comment by beornf** [comment]
LGTM thanks.

**Pull request merged**